src/composables/pendingMultisigTransaction.ts

```diff
@@ -18,12 +30,16 @@ export function usePendingMultisigTransaction({ store }: IDefaultComposableOptio
   const { fetchActiveMultisigTx } = useMultisigTransactions({ store });
   const { topBlockHeight } = useTopHeaderData({ store });
   const { accounts } = useAccounts({ store });
+  const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
+  const latestMultisigAccountTransaction = ref<ITransaction | null>(null);
 
   async function assignPendingMultisigTx() {
-    if (activeMultisigAccount.value) {
+    if (activeMultisigAccount.value?.txHash) {
```

Comment:
This conditon will be obsolete after https://github.com/aeternity/superhero-wallet/pull/1831will be merged, since the `fetchActiveMultisigTx` already conatining such condition.